metaci/build/management/commands/tests/test_run_build.py

```diff
@@ -0,0 +1,60 @@
+import os
+from unittest import mock
+
+import pytest
+from django.conf import settings
+from django.test import TestCase
+
+from metaci.build.management.commands.run_build import Command
+from metaci.build.models import Build
+from metaci.conftest import (
+    BranchFactory,
+    OrgFactory,
+    PlanFactory,
+    PlanRepositoryFactory,
+    RepositoryFactory,
+    UserFactory,
+)
+
+
+@mock.patch("metaci.build.management.commands.run_build.scratch_org_limits")
+@mock.patch("metaci.build.management.commands.run_build.run_build")
+@pytest.mark.django_db
+class TestRunBuild(TestCase):
+    # def test_run_build_scratch(self, run_build):
+    #     self._testrun_build(run_build, True)
+
+    def test_run_build_persistent(self, run_build, scratch_org_limits):
+        self._testrun_build(run_build, scratch_org_limits, False)
+
+    def test_run_build_persistent_(self, run_build, scratch_org_limits):
+        self._testrun_build(run_build, scratch_org_limits, True)
```

Comment:
Why does this test case have persistent in the name but sets org_is_scratch to True?